chaosdatadog/metrics/utils.py

```diff
@@ -1,6 +1,28 @@
 import re
 
+COMPARISON_VALUES = [">", "<", ">=", "<=", "==", "!="]
+
 
 def extract_metric_name(query):
     pattern = r"(?::|^)([^{}:]+)(?:{|$)"
-    return match.group(1) if (match := re.search(pattern, query)) else None
+    match = re.search(pattern, query)
+    return match[1] if match else None
+
+
+def check_comparison_values(comparison):
+    if comparison not in COMPARISON_VALUES:
+        raise ValueError(
```

Comment:
Not that if you want to mark the activity as failed, I would suggest raisong `ActivityFailed` instead.

Reply:
It's because this raise is inside utils.py generic functions but then of this I used ActivityFailed inside the probes.py

```
     try:
          check_comparison_values(comparison)
     except ValueError as e:
          raise ActivityFailed(e)
```
The output when use check_comparison_values and the metod raise a exception is an activity failed.

```
[2023-06-07 14:20:20 INFO] Probe: deployment doesn't have 5xx errors
**[2023-06-07 14:20:20 ERROR]   => failed: chaoslib.exceptions.ActivityFailed: Invalid value. Expected one of: '>', '<', '>=', '<=', '==', '!='**
[2023-06-07 14:20:20 WARNING] Probe terminated unexpectedly, so its tolerance could not be validated
[2023-06-07 14:20:21 CRITICAL] Steady state probe 'deployment doesn't have 5xx errors' is not in the given tolerance so failing this experiment
[2023-06-07 14:20:21 INFO] Experiment ended with status: failed
        
```